### ISSUE TITLE
How can the network be successfully trained?

### ISSUE THREAD

**Issue by Amazing6312** [Lauraxy/Multi_Modal_Registration#3]
Hello, this is my first attempt at GAN-based image registration. I have been running your model recently and have not been successfully trained. The data set uses BraTS2018 in the paper. The data is normalized, and the network parameters use the recommended values in the paper. The training has not been successful. Is there any problem with my training strategy? Can you provide the correct training experience? What is the general performance of the loss function for successful network training? Thank you
